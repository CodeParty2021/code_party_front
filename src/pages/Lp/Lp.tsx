import React from "react";
import { Background } from "pages/Event/components/Background";
import { Link } from "react-router-dom";
type Prop = {};

export const Lp: React.FC<Prop> = () => {
  return (
    <div id="overlay">
      <div id="content">
        <p>宣伝用のページです。</p>
        <Link to="/">タイトル画面に進む(Topページ)</Link>

        <Background />
      </div>
    </div>
  );
};
