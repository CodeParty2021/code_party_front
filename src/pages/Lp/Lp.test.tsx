import React from "react";
import { shallow } from "enzyme";
import { Lp } from "./Lp";
describe("<Load />", () => {
  it("auth snapshot test", () => {
    const wrapper = shallow(<Lp />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
