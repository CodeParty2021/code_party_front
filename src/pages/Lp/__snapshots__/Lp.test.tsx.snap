// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Lp /> snapshot test 1`] = `
Array [
  <div
    id="overlay"
  >
    <div
      id="content"
    >
      <p>
        宣伝用のページです。
      </p>
      <Link
        to="/"
      >
        タイトル画面に進む(Topページ)
      </Link>
      <Styled(StarBackground) />
    </div>
  </div>,
]
`;
